packages/atomic/src/index.html

```diff
@@ -106,6 +107,68 @@
         margin: 10px 0;
       }
 
+      @media only screen and (max-width: 1024px) {
+        atomic-search-interface {
+          grid-template-rows: 16px 50px 16px auto 1fr;
+          grid-template-columns: minmax(0, 100%);
+        }
+
+        atomic-facet-manager,
+        .header,
+        atomic-search-box,
+        .results-container {
+          grid-column: 1 / auto;
```

Comment:
instead of using `grid-column` and `grid-row`, perhaps we could use `area`? this may even allow reordering.